src/main/java/com/easypost/model/Report.java

```diff
@@ -44,8 +48,33 @@ public static Report create(final Map<String, Object> params, final String apiKe
         wrappedParams.put("start_date", params.get("start_date"));
         wrappedParams.put("end_date", params.get("end_date"));
 
+        List<String> columns = new ArrayList<>();
+        List<String> additionalColumns = new ArrayList<>();
+
+
+        if (params.containsKey("columns")) {
+            columns = (ArrayList<String>) params.get("columns");
+        }
+
+        if (params.containsKey("additional_columns")) {
+            additionalColumns = (ArrayList<String>) params.get("additional_columns");
+        }
+
         String type = (String) params.get("type");
-        return request(RequestMethod.POST, reportURL(type), wrappedParams, Report.class, apiKey);
+        String url = reportURL(type) + "?";
+
+        for (int i = 0; i < columns.size(); i++) {
+            url += "column[]=" + columns.get(i) + "&";
```

Comment:
Please only construct the url params inside of the above if block. Only if the columns and additional columns keys are present do we want to alter the URL